parser/src/cfg/defcfg.rs

```diff
@@ -917,6 +926,19 @@ pub fn parse_dev(val: &SExpr) -> Result<Vec<String>> {
     })
 }
 
+fn parse_cfg_val_u64(expr: &SExpr, label: &str) -> Result<u64> {
```

Comment:
This code should be deleted and instead use u16. Arguably even limiting to u8 is reasonable for this use case; a debounce time of 250ms is already likely a terrible user experience, but there is value in using existing convention and reducing binary size somewhat by just reusing u16 parsing which already exists.